Tools/Scripts/webkitpy/common/system/executive.py

```diff
@@ -391,17 +391,20 @@ def run_command(self,
                              pass_fds=pass_fds)
         with process:
             if not string_to_communicate:
-                output = process.communicate()[0]
+                output = process.communicate(timeout=Timeout.difference())[0]
             else:
-                output = process.communicate(string_utils.encode(string_to_communicate, encoding='utf-8'))[0]
+                output = process.communicate(string_utils.encode(string_to_communicate, encoding='utf-8'), timeout=Timeout.difference())[0]
 
             # run_command automatically decodes to unicode() and converts CRLF to LF unless explicitly told not to.
             if decode_output:
                 output = string_utils.decode(output, encoding=self._child_process_encoding()).replace('\r\n', '\n')
 
             # wait() is not threadsafe and can throw OSError due to:
             # http://bugs.python.org/issue1731717
-            exit_code = process.wait()
+            exit_code = process.wait(timeout=Timeout.difference())
```

Comment:
This should be a no-op, no?

`Popen.communicate` itself waits for the process to terminate or raises `TimeoutExpired`, so we're not gonna get a different result here.